src/data/scripts/generate_historic_stats.py

```diff
@@ -0,0 +1,144 @@
+"""
+to run this script
```

Comment:
Can you make this doc block more explanatory? Take a look at some of the other files, but ideally it should let you know what data it ingests and what files it creates, and why.

**You should also write tests for your Python file!**